__tests__/auth.js

```diff
@@ -77,6 +95,40 @@ describe('Authentication', () => {
     expect(response.json).toBeCalledWith(profile);
   });
 
+  test('#login - createHomeDirectory file', async () => {
+    request.setBody({username: 'jest', password: 'jest'});
+
+    await auth.login(request, response);
+    request.fields = {
+      path: 'home:/.desktop/.shortcuts.json',
+    };
+
+    const result = await filesystem.request('exists', request);
+    expect(result).toBe(true);
+  });
+
+  test('#login - createHomeDirectory directory', async () => {
+    request.setBody({username: 'jest', password: 'jest'});
+
+    const dirpath = path.resolve(
+      core.configuration.root,
+      'homeDirFolder'
+    );
+    core.configuration.vfs.home.template = dirpath;
+
+    await auth.login(request, response);
+
+    const files = await fs.readdir(dirpath);
```

Comment:
Since you're copying from a known set of files you can make some assumptions here.

Just read the known file you've added and ensure the content matches (and not that it just exists).